openfe/protocols/openmm_afe/equil_solvation_afe_method.py

```diff
@@ -404,8 +404,9 @@ def _default_settings(cls):
             solvent_system_settings=SystemSettings(),
             vacuum_system_settings=SystemSettings(nonbonded_method='nocutoff'),
             alchemical_settings=AlchemicalSettings(),
+            lambda_settings=LambdaSettings(),
```

Comment:
Would it be reasonable here to change the default to match what we usually run as a default when doing AHFEs, i.e. 14 windows?

Reply:
Ah yes, had changed that in the other PR, but not here, added this now!